docs/29_segmentation_tutorial.md

```diff
@@ -0,0 +1,250 @@
+    <img className="max-h-[500px]" src="/img/tutorials/flat_mesh.png"/>
+  </div>
+  <figcaption className="mt-1">Figure 7. UV parametrization of the vertices of the mesh displayed in Figure 6.</figcaption>
+</figure>
+
+The mesh, along with its UV parametrization, are finally saved in a `.obj` file.
+
+#### Rendering
+It is extremely important that the obtained UV coordinates are still real numbers. This means that between points there is a lot of empty space!
+In order to go back to a "pixel-style" representation, we have to convert the plot in Figure 7 to a pixelized image. Notice that in Figure 7 we displayed axes with ticks, this was not a mistake.
+
+##### Per-pixel-map and layers
+For every integer couple (x,y) in the figure we insert a new point. These coordinates (x,y) will be the integers' positions (i,j) of pixels in the rendered image. Eventually, we will end up with a number of points way higher than that of the vertices of the mesh. We will identify in which triangle of the triangular mesh (in 2D) these points fall and compute their `barycentric coordinates`: three coefficients that allow to define the position of every point in a triangle as a linear combination of the positions of the vertices. In Figure 8 we show a red point whose position will be `0.2 * vertex1 + 0.3 vertex2 + 0.5 vertex3`.
+
+<figure className="text-center mx-auto">
+  <div className="flex flex-wrap justify-center mx-auto">
+    <img className="max-h-[400px] m-2" src="/img/tutorials/barycentric.png"/>
+  </div>
+  <figcaption className="mt-1">Figure 8. A red point in a triangle represented with its barycentric coordinates.</figcaption>
+</figure>
+
+Exploiting the barycentric coordinates, we can then map back every additional point that we inserted in the UV map to its alleged position in 3D space.
+This step is called obtaining a *per-pixel-map* and it's performed in `Thaumato Anakalyptor` using a recently merged community fork [^7].
+
+Why do we need the 3D positions of the new points? Remember that during the first step, *Node Placement*, we totally forgot about the density of the material obtained via the CT scan (its original color) and started working on a black and white mask, that later became a point cloud, and so on. All the colors displayed in Figures 3-7 were only for representation purposes.
+
+Now we need to recover that information, and to do so we need to know the 3D position of every new point that is going to become a pixel of our rendered image.
+Unfortunately, the map from integer 2D positions to 3D points will result in points with non-integer positions. What does it mean? After all, the original voxels (Figure 1) _only_ have integer positions. What "color" will these new points/pixels then have?
+
+We are going to compute the color of the new points by `trilinear interpolation` with their closest ones in the volume.
+This will allow us to obtain images with a smooth varying color.
+

```

Comment:
Here it would be good to have the raw rendered image of the segment

Reply:
Thanks!